core_backend/migrations/versions/2024_06_13_28861161249b_add_content_quota_to_user_table.py

```diff
@@ -0,0 +1,30 @@
+"""add content_quota to user table
+
+Revision ID: 28861161249b
+Revises: 29b5ffa97758
+Create Date: 2024-06-13 12:01:07.418461
+
+"""
+
+from typing import Sequence, Union
+
+import sqlalchemy as sa
+from alembic import op
+
+# revision identifiers, used by Alembic.
+revision: str = "28861161249b"
+down_revision: Union[str, None] = "29b5ffa97758"
+branch_labels: Union[str, Sequence[str], None] = None
+depends_on: Union[str, Sequence[str], None] = None
+
+
+def upgrade() -> None:
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.add_column("user", sa.Column("content_quota", sa.Integer(), nullable=True))
```

Comment:
Should this be nullable? I guess that makes migration easy but migrated users would have null and cause failures when creating new content.

Reply:
Yeah, Null means no limit in the app logic (line 105 of contents/models.py):

```
        # if content_quota is None, then there is no limit
        if content_quota is None:
            pass
```

Actually, I don't think that line is even necessary because if I handle the env var correctly as "falsy" when it's missing, Null, or False (as per your other comment) then the entire check section won't even happen.